causal_testing/testing/causal_test_outcome.py

```diff
@@ -52,14 +58,19 @@ def apply(self, res: CausalTestResult) -> bool:
             ci_high = res.ci_high() if isinstance(res.ci_high(), Iterable) else [res.ci_high()]
             value = res.test_value.value if isinstance(res.ci_high(), Iterable) else [res.test_value.value]
 
-            if not all(ci_low < 0 < ci_high for ci_low, ci_high in zip(ci_low, ci_high)):
-                print(
-                    "FAILING ON",
-                    [(ci_low, ci_high) for ci_low, ci_high in zip(ci_low, ci_high) if not ci_low < 0 < ci_high],
-                )
+            # if not all(ci_low < 0 < ci_high for ci_low, ci_high in zip(ci_low, ci_high)):
```

Comment:
If avoidable I think it would be better to just remove code rather than commenting out. It can always be found in previous versions of the framework if needed again.

Reply:
Again, you're probably right. I'm just a lazy programmer.